Feature pgps ncsdk 11580 (#5745)
nrfconnect/sdk-nrf: doc/nrf/releases/release-notes-changelog.rst

    * Removed GNSS socket API support from A-GPS and P-GPS.
    * Added support for sending data to a new bulk endpoint topic that is supported in nRF Cloud.
      A message published to the bulk topic is typically a combination of multiple messages.

>     * Fixed an issue with :kconfig:`CONFIG_NRF_CLOUD_PGPS_TRANSPORT_NONE` to ensure predictions will be properly stored.

Reviewer: ```suggestion
    * Fixed an issue with :kconfig:`CONFIG_NRF_CLOUD_PGPS_TRANSPORT_NONE` to ensure predictions are properly stored.
```

Author: I did make this change.

    * Added :c:func:`nrf_cloud_pgps_request_reset` so P-GPS application request handler can indicate failure to process the request.

>       This ensures the P-GPS library will try the request again.

Reviewer: ```suggestion
      This ensures the P-GPS library tries the request again.
```

Author: I also made this change.
